### DISCUSSION TITLE
An unloaded video resource (suspected)

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#297] [Q&A]
[https://www.biostarhandbook.com/whatisdata.html](https://www.biostarhandbook.com/whatisdata.html#:~:text=We%20went%20around%20and%20asked%20a%20bunch%20of%20people%20the%20hard%20question%20What%20is%20data%3F.)
What's This? Suspect is a [video resource](https://voicethread.com/app/player/?threadId=6822610), but I can not play it correctly

**Comment by ialbert** [comment]
Looks like the video hosting service, VoiceThread is not loading the resource. 

I have disabled the embeding until I track down why it is not loading.

thanks for reporting.
